dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -19,154 +19,160 @@
-    "is used to determine when the worker starts spilling to host "
-    "memory.  This can be a float (fraction of total device "
-    "memory), an integer (bytes), a string (like 5GB or 5000M), "
-    "and 'auto' or 0 to disable spilling to host (i.e., allow "
-    "full device memory usage). Default is 0.8, 80% of the "
-    "worker's total device memory.",
+    show_default=True,
+    help="""Specifies the size of the CUDA device LRU cache, which is used to determine
+    when the worker starts spilling to host memory; can be an integer (bytes), float
+    (fraction of total device memory), string (like ``5GB`` or ``5000M``), ``auto``,
+    or 0 to disable spilling to host (i.e., allow full device memory usage)""",
 )
 @click.option(
     "--rmm-pool-size",
     default=None,
-    help="If specified, initialize each worker with an RMM pool of "
-    "the given size, otherwise no RMM pool is created. This can be "
-    "an integer (bytes) or string (like 5GB or 5000M)."
-    "NOTE: This size is a per worker (i.e., per GPU) configuration, "
-    "and not cluster-wide!",
+    help="""If specified, initialize each worker with an RMM pool of the given size; can
+    be an integer (bytes) or string (like ``5GB`` or ``5000M``); this size is per-worker
+    (GPU) and not cluster-wide
+
+    .. note::
+        This size is a per-worker (GPU) configuration, and not cluster-wide""",
 )
 @click.option(
     "--rmm-managed-memory/--no-rmm-managed-memory",
     default=False,
-    help="If enabled, initialize each worker with RMM and set it to "
-    "use managed memory. If disabled, RMM may still be used if "
-    "--rmm-pool-size is specified, but in that case with default "
-    "(non-managed) memory type."
-    "WARNING: managed memory is currently incompatible with NVLink, "
-    "trying to enable both will result in an exception.",
+    show_default=True,
+    help="""If enabled, initialize each worker with RMM and set it to use managed
+    memory; if disabled, RMM may still be used by specifying ``--rmm-pool-size``
+
+    .. warning::
+        Managed memory is currently incompatible with NVLink; trying to enable both will
+        result in an exception.""",
 )
 @click.option(
     "--rmm-log-directory",
     default=None,
-    help="Directory to write per-worker RMM log files to; the client "
-    "and scheduler are not logged here."
-    "NOTE: Logging will only be enabled if --rmm-pool-size or "
-    "--rmm-managed-memory are specified.",
+    help="""Directory to write per-worker RMM log files to; the client
+    and scheduler are not logged here.
+
+    .. note::
+        Logging will only be enabled if ``--rmm-pool-size`` or ``--rmm-managed-memory``
+        are specified.""",
+)
+@click.option("--pid-file", type=str, default="", help="File to write the process PID")
+@click.option(
+    "--resources",
+    type=str,
+    default="",
+    help="""Resources for task constraints like ``GPU=2 MEM=10e9``; resources are
```

Comment:
Didn't really understand this input when I passed over it; is this meant to be one string i.e. `"GPU=2 MEM=10e9"`?

Reply:
AFAIU, yes. This is to provide annotations so that the application may later constrain certain parts of its execution to specific workers, see [Worker Resources](https://distributed.dask.org/en/latest/resources.html).